src/pages/iep/[iep_id].tsx

```diff
@@ -0,0 +1,76 @@
+import { trpc } from "@/client/lib/trpc";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import styles from "../../styles/Home.module.css";
+import Goals from "../../components/Goal";
+
+interface IepProps {
+  iep_id: string | string[] | undefined;
+}
+
+const Iep: React.FC<IepProps> = ({ iep_id }) => {
```

Comment:
Next.js pages receive no props unless a SSR handler is defined

```suggestion
const Iep = () => {
```

Reply:
Ah ok - good to know. Done